sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -1254,21 +1254,38 @@ class AdaptiveQueryExecSuite
   test("SPARK-34533: Eliminate left anti join to empty relation") {
     withSQLConf(
       SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> "true") {
-      withTable("emptyTestData") {
```

Comment:
Do not create `emptyTestData` anymore as we already [created one here](https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/test/SQLTestData.scala#L295). Create table again here will cause problem for the followed newly added test when trying to use `emptyTestData`.